TombEngine/Game/camera.cpp

```diff
@@ -405,6 +405,16 @@ void MoveObjCamera(GameVector* ideal, ItemInfo* camSlotId, int camMeshId, ItemIn
 	}
 }
 
+void RefreshFixedCamera(short camNumber)
+{
+	auto* camera = &g_Level.Cameras[camNumber];
+
+	GameVector from = GameVector(camera->Position, camera->RoomNumber);
```

Comment:
You can use `auto` here. Should also be named `origin` like everywhere else.

Reply:
done